### DISCUSSION TITLE
[A-L] (2023/24) Lezione 11/10/2023 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#165] [Esercizi - Viaggi/Piazza]
**Esercizio 1:** Dimostrare che l'operazione di somma su $\mathbb{Q}$ è ben definita, ovvero non dipende dalla scelta dei rappresentati delle classi.
**Definizione** $+:= [(a,b)]+[(a',b')]=[(ab'+b'a,bb')]$

Quindi dimostrare: $(a,b) \rho (c,d) \land (a',b')\rho (c',d') \Rightarrow (ab'+ba',bb') \rho (cd'+dc',dd')$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/8ec6fbc4-0388-43d7-9c38-33969dbad7f0)
